src/utils/signaling.js

```diff
@@ -1430,8 +1431,12 @@ Signaling.Standalone.prototype.processRoomEvent = function(data) {
 Signaling.Standalone.prototype.processRoomMessageEvent = function(token, data) {
 	switch (data.type) {
 		case 'chat':
-		// FIXME this is not listened to
-			EventBus.emit('should-refresh-chat-messages')
+			if ('comment' in data.chat) {
+				EventBus.emit('signaling-message-received', { token, message: data.chat.comment })
```

Comment:
Follow-up: this is coming faster than your response; e.g. therefore added reaction doesn't count as yours

Reply:
For the history, this is what coming from federated room. We can ask @fancycode whether it's possible to translate token (to roomid) on HPB side?

<img width="759" height="493" alt="image" src="https://github.com/user-attachments/assets/f651c103-9324-4203-8d71-8c334d6d969d" />